setup.cfg

```diff
@@ -15,7 +15,11 @@ process-timeout=300
 #pdb=1
 
 [tool:pytest]
-norecursedirs = .git conf node_modules test_root cms/envs lms/envs
+DJANGO_SETTINGS_MODULE = lms.envs.test
+addopts = --nomigrations --reuse-db --durations=20
+norecursedirs = .* *.egg build conf dist node_modules test_root cms/envs lms/envs
+python_classes =
```

Comment:
@jmbowman is there a reason you excluded classes? This forces us to continue using unittest-style classes despite not needing them.

Reply:
I think the reason was that we have some mix-ins which look like tests but contain no actual test cases.  nose was glossing over these because nothing failed, but pytest counts them as failures because it considers the lack of actual test cases in a test class to be an error.  This was simpler than going through to rename and refactor a bunch of code in multiple modules.

Standard practice with pytest is just to use test functions and fixtures whenever possible, although the transaction+savepoints magic in django.test.TestCase is a compelling reason to keep using that in many cases.  Did you have a particular use case in mind for a non-TestCase test class?

Reply:
I wanted to ditch `unittest` for the tests in https://github.com/edx/edx-platform/pull/16451.

Reply:
@clintonb It looks like you could replace that test class with 3 test functions and a setup fixture, if you'd like; what would you gain by just switching the base class to `object`?

Reply:
Good point. I am still too dependent on classes.